### DISCUSSION TITLE
Tool to test which actions will be performed with a given configuration and pull request

### DISCUSSION THREAD

**Discussion by bahrmichael** [renovatebot/renovate#17228] [Suggest an Idea]
We've recently started using Renovate and there are a lot of configuration options that we apply. We found it difficult sometimes to understand if a configuration works as expected when only having the debug logs of the (non-self) hosted bot for GitHub, and the pull requests it creates.

We had a great experience with [Mergify and its config editor](https://dashboard.mergify.com/), where we can paste a config and provide a PR number. It then shows us on the right hand side which rules are met, and which aren't. This gives us a very fast feedback loop and lets us quickly complete the setup. With Renovate we sometimes wait an hour or more until we see which actions the bot takes.

<img width="1438" alt="Screenshot 2022-08-17 at 10 15 08" src="https://user-images.githubusercontent.com/1830132/185069386-7d52f4d7-f91e-43f9-b625-9579be1558ef.png">

I think a tool for Renovate where I can paste or link a config file, and provide a pull request number would be immensely helpful to quickly get started. What do you think? Happy to draft more details in an issue if this discussion goes well :)

**Comment by HonkingGoose** [comment]
Yeah the onboarding of Renovate can be a bit tricky. You're right that we don't really have interactive tools to help you with this.

Brainstorming some more, it would be nice to have a site where you can:

- Paste your `renovate.json` config file
- Check if the config is valid
  - Automatically update old config option names via "config migration"
- Click on config option names to see the docs in a new tab
- Get a summary of what will happen

**Reply by aisamu** [reply]
* View the resolved config and where things were sourced from.
   > Extensions, presets, local overrides... akin to `git config --list --show-origin`!
